README.md

```diff
@@ -1,7 +1,31 @@
 # ttt
 
 An implementation of [tic-tac-toe](https://en.wikipedia.org/wiki/Tic-tac-toe) in C,
-featuring an AI powered by the [negamax](https://en.wikipedia.org/wiki/Negamax) algorithm.
+featuring an AI powered by the [negamax](https://en.wikipedia.org/wiki/Negamax) algorithm and the [TD learning](https://en.wikipedia.org/wiki/Temporal_difference_learning) algorithm.
+
+## Run
+If you want to play with negamax AI:
+Build the game:
+```bash
+$ make
+```
+and run:
+```bash
+$ ./ttt
+```
+If you want to play with TD learning AI:
+Train the model first:
+```bash
+$ make train
+```
```

Comment:
Adding instructions for running the training process may help make the documentation clearer.